api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -206,6 +207,15 @@ message FilterChain {
   // name is provided, Envoy will allocate an internal UUID for the filter chain. If the filter
   // chain is to be dynamically updated or removed via FCDS a unique name must be provided.
   string name = 7;
+
+  // [#not-implemented-hide:] Whether the filter chain will be built on demand or not.
```

Comment:
Could you add several extra lines: if the filter chain is ondemand, the filter chain is not blocking listener update, and the filter chain initialization will be postponed until the first incoming traffic

Reply:
Great idea! I will added those descriptions.